microsoft-365/admin/add-users/let-users-reset-passwords.md

```diff
@@ -49,7 +49,8 @@ If you found this video helpful, check out the [complete training series for sma
 These steps turn on self-service password reset for everyone in your business.
   
 ::: moniker range="o365-worldwide"
-1. In the Microsoft 365 admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>, in the left navigation pane, go to the **Settings** \> **Org settings** page.
+
+1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339), go to the **Settings** > **Org settings** page.
```

Comment:
@cmcatee-MSFT  - sure, I will update the change, I am curious to know what is difference in both.

Reply:
@PrasadKasireddy The difference is that using the HTML formatting forces the link to open in a new tab so the reader doesn't lose the page they're currently on. For deep-links that go into the product (in this case, the Microsoft 365 admin center), we use this method, rather than just using standard markdown for inline links.